### ISSUE TITLE
Yo on Ubuntu 20, not working after deployment

### ISSUE THREAD

**Issue by uhlhosting** [yeoman/yo#674]
## Type of issue

----

## My environment

* *OS version/details*: `Ubuntu 20.04`
* *Node version:* `v14.16.1` (run `node --version` in your terminal)
* *npm version:* `7.16.0` (run `npm --version` in your terminal)
* *Version of yo :* `x.x.x` (run `yo --version` in your terminal)

## Current behavior
I installed it as suggested, nothing is found no binary and yo is simply not called no matter with what user, root, or any local user, alto was globally installed.

## Steps to reproduce the behavior
Get clean KVM ubuntu 20.04 deploy nodejs npm and use npm to install yo, see all the warnings due to missing updates and yeah, who cares, yadayada...

No command is registered nor found out of the box. I was hoping the binaries will be available. Tried npm and yarn deployment, both end succesfully yet none of them find the yo binary in shell..

## Command line output

```
<!-- zulip@chat:~$ sudo -H npm install --global yo
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated cross-spawn-async@2.2.5: cross-spawn no longer requires a build toolchain, use it instead
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142

changed 712 packages, and audited 713 packages in 6s

50 packages are looking for funding
  run `npm fund` for details

7 high severity vulnerabilities

To address all issues, run:
  npm audit fix

Run `npm audit` for details.
zulip@chat:~$ yo --version
yo: command not found
zulip@chat:~$ 
 -->
```

**Comment by Daksh-10** [comment]
Yes same, I was working on windows before, but I changed path for program files for many apps so, only I can use them, but now I was not able to execute commands from VS Code myself because I forgot, and then now it is also not working on Ubuntu. : ). Did you find any solutions to work them on ubuntu?

**Comment by jozefizso** [comment]
Hi @uhlhosting, please ensure your `PATH` is setup correctly. It must include path to your global npm installation where the tools are.

```shell
echo "$(npm prefix -g)/bin"
```

See https://docs.npmjs.com/cli/v11/commands/npm-prefix

This is not issue with the `yo` tool and you must ensure your machine is configured correctly.

**Comment by JoshuaKGoldberg** [comment]
+1 to https://github.com/yeoman/yo/issues/674#issuecomment-2564743144 ☝ , this isn't an issue with `yo`. Yeoman / `yo` supports Mac, Ubuntu & other common Linux distributions supported by Node.js, Windows, etc.
